tests/test_biophysical/test_animal/test_stocker/test_beef_stocker_constants.py

```
@@ -0,0 +1,124 @@
+import pytest
+
+from RUFAS.biophysical.animal.animal_module_constants import AnimalModuleConstants
+
+# ---------------------------------------------------------------------------
+# Exact values — pinned to NRC 2016 Ch.10-12 source
+# ---------------------------------------------------------------------------
+
+
+@pytest.mark.unit
+@pytest.mark.parametrize(
+    "attr, expected",
```

Ruff (0.15.21)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)